app/components/form_status_tag_component/view.rb

```diff
@@ -1,17 +1,21 @@
 module FormStatusTagComponent
   class View < ViewComponent::Base
-    attr_accessor :status
-
-    def initialize(status: "DRAFT")
+    def initialize(status: :draft)
       super
-      @status = status.upcase
+      @status = status.to_sym
     end
 
     def status_colour
       {
         draft: "purple",
         live: "blue",
-      }[status.downcase.to_sym]
+      }[@status]
+    end
+
+    def status_text
+      # i18n-tasks-use t('form_statuses.draft')
+      # i18n-tasks-use t('form_statuses.live')
```

Comment:
```suggestion
```

Reply:
Unfortunately we need these [hints](https://github.com/glebm/i18n-tasks#fine-tuning) :/ I found that without them the `i18n-tasks missing` test didn't warn us if one of these translations was deleted.